src/execution_plan/ops/shared/traverse_functions.c

```
@@ -60,20 +62,16 @@ static GRAPH_EDGE_DIR _Traverse_SetDirection
 
 	// push down transpose operations to individual operands
 	AlgebraicExpression_PushDownTranspose(ae);
-	AlgebraicExpression *parent = NULL;
 	AlgebraicExpression *operand = NULL;
 
 	// locate operand representing the referenced edge
-	bool located = AlgebraicExpression_LocateOperand(ae, &operand, &parent,
+	bool located = AlgebraicExpression_LocateOperand(ae, &operand,
 			e->src->alias, e->dest->alias, e->alias, NULL);
 	ASSERT(located == true);
```

cppcheck

[error] Null pointer dereference

(nullPointer)